cmd/beaker/image/create.go

```
@@ -125,14 +125,16 @@ func Create(
 	if err != nil {
 		return "", errors.WithStack(err)
 	}
-	defer r.Close()
-
 	// Display push responses as the Docker CLI would. This also translates remote errors.
 	var stream io.Writer = os.Stdout
 	if opts.Quiet {
 		stream = ioutil.Discard
 	}
 	if err := jsonmessage.DisplayJSONMessagesStream(r, stream, 0, false, nil); err != nil {
+		r.Close()
```

Any time we don't handle an error, I think we should get into the habit of documenting why. e,g. here do a `_ = r.Close()` and add a comment saying the error doesn't affect the handling here.

---

Also, the errcheck linter forces this: https://github.com/kisielk/errcheck

---

Made ignoring the error explicit. I don't think a comment is useful here.

---

That's fine.